File: valtozo.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
a = int(input("Kérem az a változót: "))
b = int(input("Kérem a b változót: "))

a = b - a
b = b - a
a = b + a
print "Kivonás-összeadással:"
print "a={}".format(a)
print "b={}".format(b)

a = a * b
b = a / b
a = a / b
print "Szorzás-osztással:"
print "a={}".format(a)
print "b={}".format(b)

File: valtozo.c
#include <stdio.h>

int main(void)
{
	int a,b;
	printf("Kérem az a változót: ");
	scanf("%d", &a);
	printf("Kérem a b változót: ");
	scanf("%d", &b);

	a = b - a;
	b = b - a;
	a = b + a;
	printf("Kivonás-összeadással:\na=%d ", a);
	printf("\nb=%d ", b);
	printf("\n");

	a = a * b;
	b = a / b;
	a = a / b;
	printf("Szorzás-osztással:\na=%d ", a);
	printf("\nb=%d ", b);
	printf("\n");
}
